### ISSUE TITLE
provisioning error on debian 8

### ISSUE THREAD

**Issue by cpuguy83** [docker/machine#1866]
Provisioning on debian jessie yields an error, though everything does seem to work just fine.
```
$ docker-machine create -d digitalocean \
  --engine-install-url https://experimental.docker.com \
  --digitalocean-image "debian-8-x64" \
  swarm-demo-1
Creating SSH key...
Creating Digital Ocean droplet...
SSH cmd error!
command: sudo systemctl enable docker
err    : exit status 1
output : Synchronizing state for docker.service with sysvinit using update-rc.d...
Executing /usr/sbin/update-rc.d docker defaults
Executing /usr/sbin/update-rc.d docker enable
Failed to execute operation: File exists
```

**Comment by nathanleclaire** [comment]
cc @ehazlett

**Comment by ehazlett** [comment]
Duplicate of #1705.  We will get this fixed for 0.5
